### ISSUE TITLE
doc request: how to install

### ISSUE THREAD

**Issue by jwaldmann** [adamwalker/sylvan-haskell#1]
I guess we need to first install sylvan, then `cabal install` this package here? (since `stubs.c` references `sylvan.h`)

**Comment by adamwalker** [comment]
Yes, install Sylvan first. But, it still probably wont work. See: https://github.com/adamwalker/sylvan-haskell/issues/2.
